fix: Copy link from 3dots - EXO-80141

mkrout opened the pull request:
This commit will fix the copy link to be relative to documents type and permissions.

mkrout committed:
fix: Copy link from 3dots - EXO-80141 (#1508)

This commit will fix the copy link to be relative to documents type and permissions.

boubaker commented on documents-webapp/src/main/webapp/vue-app/documents/components/body/actions/CopyLinkMenuAction.vue:
```diff
@@ -26,31 +26,28 @@ export default {
+      input.value = path;
+      document.body.appendChild(input);
+      input.select();
+      document.execCommand('copy');
```
Just for my information, why not using `navigator.clipboard.writeText` knowing that [document.execCommand](https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand) is deprecated ?

mkrout replied:
yes, I just missed that, it was the case, we check if  'navigator.clipboard.writeText' is available for the current browser otherwise we use the execCommand I'll restore that. thanks

mkrout committed:
fix: Copy link from 3dots - EXO-80141
This commit will fix the copy link to be relative to documents type and permissions.
